pyproject.toml

```diff
@@ -189,73 +268,118 @@ src = ["rockcraft", "tests"]
 extend-exclude = [
     "docs",
     "__pycache__",
-    "rockcraft/_version.py"
+    "rockcraft/_version.py",
+    "tests/spread",
```

Comment:
question: what was the issue? Is that because this folder holds python files we do not want to lint?